packages/client/src/components/Modals/GrantDetails.vue

```diff
@@ -164,20 +164,30 @@ export default {
       agency: 'users/agency',
       selectedAgencyId: 'users/selectedAgencyId',
       agencies: 'agencies/agencies',
+      currentTenant: 'users/currentTenant',
       users: 'users/users',
       interestedCodes: 'grants/interestedCodes',
+      user: 'users/loggedInUser',
     }),
     alreadyViewed() {
       if (!this.selectedGrant) {
         return false;
       }
-      return this.selectedGrant.viewed_by_agencies.find((viewed) => viewed.agency_id.toString() === this.selectedAgencyId);
+      return this.selectedGrant.viewed_by_agencies.find(
+        (viewed) => viewed.agency_id.toString() === this.selectedAgencyId,
+      );
+    },
+    shouldShowSpocButton() {
+      if (!this.currentTenant.uses_spoc_process) return false;
+      return true;
     },
     interested() {
       if (!this.selectedGrant) {
         return false;
```

Comment:
nit: let's make this return `undefined` so the return type is consistent with what `find` could return below.